bottlerocket-settings-models/modeled-types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bottlerocket-modeled-types"
-version = "0.6.0"
+version = "0.7.0"
```

Comment:
The `modeled-types` didn't actually change -- we want to bump the version for:
* The kubernetes settings extension
* The `settings-models` package.

That should be it.

Reply:
Oh. I thought PR #64 changed the modeled-type and would need the version bump as a result. 

I'll updated the settings extension.

Reply:
You're right! My mistake.